applications/launchpad_v2/src/containers/Dashboard/ExpertView/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { useDispatch, useSelector } from 'react-redux'
+import { toggleExpertView } from '../../../store/app'
+import { selectExpertView } from '../../../store/app/selectors'
+
+const ExpertView = () => {
+  const dispatch = useDispatch()
+  const expertView = useSelector(selectExpertView)
+
+  return (
+    <div>
+      <p style={{ color: '#fff' }}>Expert View</p>
```

Comment:
color -> theme

Reply:
Just the temporary content of the Expert view drawer. Eventually, it has to be removed.